src/main/java/it/gov/pagopa/payhub/activities/activity/ingestionflow/organization/OrganizationIngestionActivityImpl.java

```diff
@@ -54,7 +56,7 @@ protected OrganizationIngestionFlowFileResult handleRetrievedFiles(
       return csvService.readCsv(filePath,
           OrganizationIngestionFlowFileDTO.class, (csvIterator, readerException) ->
               organizationProcessingService.processOrganization(csvIterator, readerException,
-                  ingestionFlowFileDTO, workingDirectory));
+                  ingestionFlowFileDTO, workingDirectory), ingestionFlowFileDTO.getFileVersion());
```

Comment:
This flow has not version set on csv DTO (neither ingestionFlowFileDTO.getFileVersion() is probably set)
```suggestion
                  ingestionFlowFileDTO, workingDirectory), null);
```